cli/cli/out/post_process_error_test.go

```diff
@@ -9,12 +9,12 @@ import (
 func TestRemoveFilePathFromErrorMessage(t *testing.T) {
 	stacktraceErr := createDummyStackTraceWithNonEmptyMsg()
 	errorClean := removeFilePathFromErrorMessage(stacktraceErr.Error())
-	expectedValue := "this is propagated error\n Caused by: Error: this is base error\n "
+	expectedValue := "this is propagated error\nCaused by: Error: this is base error"
```

Comment:
I think having a `\n` at the end was nice. If we ever print something after the error, it will be on the same line as the last stacktrace line, and will be very difficult to parse for a human. Ca we re-add this?

Reply:
AND, I think the blank space before `Caused by ` was nice as well as it provided some kind of indentation. I understand why it got away, would still be nice if we could get it back